provision-contest/ansible/grafana.yml

```diff
@@ -21,3 +21,5 @@
       when: GRAFANA_MONITORING
     - role: grafana
       tags: grafana
+    - role: script_server
```

Comment:
Not sure why this is part of the grafana playbook, but we can change that at a later time.

Reply:
the grafana box is most of the time the special machine which runs everything. I can decouple (later) if needed.